changelogs/fragments/56418-S3-buckets-path-change

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- Implement new supported path for bucket access (https://github.com/ansible/ansible/issues/56124)
```

Comment:
Changelog fragments aren't needed for test changes.

Reply:
K